crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs

```diff
@@ -126,11 +126,26 @@ fn compare_classes(a: &ClassInfo, b: &ClassInfo) -> Ordering {
 }
 
 /// Sort the given class string according to the given sort config.
```

Comment:
Please update the comment of the function. You updated its logic, and it should be reflected in its documentation too.

Reply:
@hangaoke1 can you address this comment?

Reply:
done